module/utils/actor-generator.js

```diff
@@ -70,10 +71,16 @@ export class ActorGenerator {
             if(game.ptu.cache.capabilities) allCapabilities = game.ptu.cache.capabilities;
             else throw 'Capabilities not cached, please provide moves.';
         }
-        return this.PrepareEvolution().PrepareGender().PrepareStats(statMethod).PrepareMoves(allMoves).PrepareAbilities(allAbilities).PrepareCapabilities(allCapabilities);
+
+        let stat_rng = 0.1;
+        if(stat_randomness)
+        {
+            stat_rng = Number(stat_randomness/100);
+        }
+        return this.PrepareEvolution(prevent_evolution).PrepareGender().PrepareStats(statMethod, stat_rng).PrepareMoves(allMoves).PrepareAbilities(allAbilities).PrepareCapabilities(allCapabilities).PrepareShiny(shiny_chance);
```

Comment:
Unnecessary variables again, could just make use of stat_randomness, also we have a default value in the settings so why do we have hardcoded 0.1

Will fix this in local as it won't let me select all lines at ones on the github website.